tests/foreman/ui/test_oscapcontent.py

```diff
@@ -37,9 +34,9 @@ def test_positive_create(self):
         @Assert: Whether creating  content for OpenScap is successful.
         """
         with Session(self.browser) as session:
+            set_context(session, org=ANY_CONTEXT['org'])
```

Comment:
For Sat6.2 , we shouldn't be running this with "Any context". It should be running under an Org.